pkg/volume/host_path/host_path.go

```diff
@@ -350,7 +351,9 @@ type fileTypeChecker struct {
 }
 
 func (ftc *fileTypeChecker) Exists() bool {
-	return ftc.mounter.ExistsPath(ftc.path)
+	exists, err := ftc.mounter.ExistsPath(ftc.path)
+	glog.Infof("path %s got %v, %v", ftc.path, exists, err)
```

Comment:
Increase log level? This looks like debugging logs

Reply:
removed whole log line